mkosi

```diff
@@ -22,6 +22,9 @@ from enum import Enum
 
 __version__ = '1'
 
+if sys.version_info[1] < 5:
```

Comment:
They are sorted in "dictionary" order, i.e. pairs of elements in the same position are compared, and if they are all equal, the longer sequence is later. This comparison is a standard Python idiom, in particular used quite often for versions ;)